2_Operations/work-process.md

```diff
@@ -6,9 +6,11 @@
 * Every **Monday and Thursday** morning we hold a daily standup meeting:
    * 10:00 CE(S)T for the whole company in the Niteo Zoom room
    * followed immediately by separate standups for WooCart, Easy Blog Networks and Kafkai, in their own Zoom rooms
-* Every **Tuesday and Wednesday** morning we only have project standup meetings:
-   * 10:00 CE(S)T separate standups for WooCart, Easy Blog Networks and Kafkai, in their own Zoom rooms
-* No standups on **Fridays**. This way Niteans can either be more productive (focus on their Stories) or take a day off (see [Vacation policy](https://github.com/niteoweb/handbook/blob/master/5_People/benefits.md#vacation)). Same as any other vacation off, make sure that your backup Nitean is available to handle any potential issues.
+
+We're straying away from daily standups since there is visible increase in prodcutivity when we started implementing no standup days. Regular Zoom meetings are still happening amongst Niteans when and where needed. 
+
+Niteans are also encouraged to take Fridays off for a long weekened. Same as any other vacation off, make sure that your backup Nitean is available to handle any potential issues (see [Vacation policy](https://github.com/niteoweb/handbook/blob/master/5_People/benefits.md#vacation)).
```

Comment:
I'm actually for removing this part since it made sense when we only had one no standup day. No strong opinion though.

Reply:
I would leave it.

Reply:
Since the section is `Schedule` I'd like it to mention bi-weekly retros/plannings too.